java/arcs/core/storage/StorageProxy.kt

```diff
@@ -107,7 +107,7 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperationAtTime, T>(
         if (runNow) {
             coroutineScope {
                 launch {
-                    action
+                    action()
```

Comment:
Just a note, but sometime in the next 2 days, we need to fix this so that the storageProxy queues these calls and runs them on a single-threaded drain to dispatch, and pauses all storage proxies while its happening, something like:

1) each arc on each ArcHost creates a HandleManager
2) when you're about to run a callback, pause all storageproxies (they must queue incoming proxy messages)
3) run the callback
4) once the callback queue is drained, you can process ProxyMessages again

At least I think that's roughly the semantic. From the time a callback is executing, all incoming proxy messages must not mutate any of the storageproxies, and we want to singlethread everything, so invoking the callbacks on a single threaded dispatcher is probably needed.